core-bundle/tests/Routing/Route404ProviderTest.php

```diff
@@ -167,17 +253,41 @@ public function testCreatesTwoRoutesWithLocale(): void
         $route = $routes['tl_page.17.error_404.locale'];
         $this->assertInstanceOf(Route::class, $route);
         $this->assertSame('.*', $route->getRequirement('_url_fragment'));
-        $this->assertSame('de', $route->getRequirement('_locale'));
         $this->assertTrue($route->getOption('utf8'));
         $this->assertSame('example.com', $route->getHost());
         $this->assertSame(['https'], $route->getSchemes());
-        $this->assertSame('/{_locale}/{_url_fragment}', $route->getPath());
+        $this->assertSame('/de/{_url_fragment}', $route->getPath());
     }
 
+//    public function testLocale()
```

Comment:
Uncomment or remove?

Reply:
Removed in 164291bb170c7ec3450f87083c41ed0e03176c99